File: MPI/mpicheck/tests/functions/2.0/MPI_File_delete.f90

        
        program main
        use mpi
        
        CHARACTER*(10) var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_file_delete(var_0, var_1, var_2)
       call pmpi_file_delete(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_File_delete.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const char *var_0;
    MPI_Info var_1;
    int ret;
    /* calls */
    ret = MPI_File_delete(var_0, var_1);
    ret = PMPI_File_delete(var_0, var_1);
    return 0;
}
